web/src/libs/utils.ts

```diff
@@ -69,14 +69,20 @@ export function parseRepoNameFromUrl(url: string): string {
 }
 
 export function parseOrgNameFromUrl(url: string): string {
-  //url is in the format of <prefix>/repository/<org>/<repo>
+  //url is in the format of <prefix>/repository/<org>/<repo> or <prefix>/organization/<org>/<repo>
   //or <prefix>/repository/<org>/<repo>/tag/<tag>
   const urlParts = url.split('/');
   const repoKeywordIndex = urlParts.indexOf('repository');
-  if (repoKeywordIndex === -1) {
+  const orgKeywordIndex = urlParts.indexOf('organization');
```

Comment:
`indexOf` isn't a reliable solution, check the repo https://console.redhat.com/quay/repository/rh-obulatov/repository

Reply:
yes